docs/protocol/concepts/tokens.md

```diff
@@ -1,9 +1,12 @@
 ---
-sidebar_position: 10
+sidebar_position: 11
 ---
 
 # Tokens
 
+Tokens represent a digital asset that functions on a blockchain, while coins represent a cryptocurrency that can be
+exchanged and used as a medium of exchange.
+
```

Comment:
On evmos, we have three types of assets

- the native Evmos token
- ibc coins
- Ethereum type tokens, e.g. ERC20

It would be great to emphasize this here, since we often just talk about two types

```suggestion
### Native Evmos Token

```